### DISCUSSION TITLE
PrimeVue Theme Designer Gone?

### DISCUSSION THREAD

**Discussion by pointermess** [primefaces/community#727] [PrimeVue] [answered]
Hello,

a few weeks ago I used a theme designer on the website of PrimeVue. It seems to be gone and I cant find any informations about it... Does anyone know what happened to it and if it is still hosted somewhere?

Thanks a lot.

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

Due to the PrimeVue Sass Theme being open-source now, we do not provide support for PrimeVue Designer. Currently, working on a new styled mode, and with version 4 we'll introduce a new PrimeVue Designer.

Thanks for your interests,
